packages/recomponents/src/styles/theme.scss

```diff
@@ -145,8 +145,8 @@ body, .article {
 
     // Elevation
     // ------------------------------------------------
-    --elevation-100: 0 0 0 1px var(--gray-color-light), 0 1px 2px 0 var(--gray-color-light), 0 4px 4px 0 var(--mono-color-100);
-    --elevation-200: 0 0 0 1px var(--gray-color-light), 0 7px 14px 0 var(--gray-color-light), 0 3px 6px 0 var(--gray-color-light);
+    --elevation-100: 0 0 0 1px rgba(74, 74, 75, 0.09), 0 1px 2px 0 rgba(74, 74, 75, 0.15), 0 4px 4px 0 rgba(74, 74, 75, 0.07);
+    --elevation-200: 0 0 0 1px rgba(74, 74, 75, 0.09), 0 7px 14px 0 rgba(74, 74, 75, 0.15), 0 3px 6px 0 rgba(74, 74, 75, 0.07);
```

Comment:
why not vars in this case?

Reply:
I could create some `vars` from this but I want more time to think about which color I want. In the meantime, we could use this and update later on. Thanks for the review